docs/meshstack.identity-import.md

```diff
@@ -0,0 +1,6 @@
+---
+id: meshstack.ldap-identity-import
+title: LDAP Identity Import
```

Comment:
d: I am not sure whether we should mention LDAP here. Currently it is the only option we support, but in general, the Identity Import is intended to be used with multiple different "connectors"

Reply:
Yes I agree. On the roadmap it is called Group Sync and ldap is only the first protocol we support with the integration service

Reply:
Should I rename it to `Identity Import` or `Group Sync`?

Reply:
meshCustomer Group Synchronisation

Reply:
Changed.